### DISCUSSION TITLE
请问如何将一页显示不全的长算法设置为多页显示？

### DISCUSSION THREAD

**Discussion by cyzhou1221** [nuaatug/nuaathesis#60] [Q&A] [answered]
尝试了 stackoverflow  中的 [https://tex.stackexchange.com/questions/33866/algorithm-tag-and-page-break]()，发现与模板不匹配.

**Comment by yzwduck** [comment, marked as answer]
我试了链接中的解决方法 https://tex.stackexchange.com/a/33882 觉得它完全可行，不知道你说的模板不匹配具体是指编译时报错，还是输出的样式不一致。

备注：如需汉化 Algorithm 标签，请在 "global.tex" 里添加下列内容：
```latex
\def\algorithmautorefname{算法}
\makeatletter
\renewcommand{\ALG@name}{算法}
\makeatother
```

**Reply by cyzhou1221** [reply]
好滴，谢谢学长，我再试试，一探究竟！

**Reply by cyzhou1221** [reply]
已经解决啦，之前遇到问题是因为多个算法宏包共用产生了冲突！
分页算法我是用的这个宏包 [algorithmicx](https://mirror.physik.tu-berlin.de/pub/CTAN/macros/latex/contrib/algorithmicx/algorithmicx.pdf)，点击有详细教程.
